diff --git a/cmake/external/pybind11.cmake b/cmake/external/pybind11.cmake
index 86aee98..c58c54f 100644
--- a/cmake/external/pybind11.cmake
+++ b/cmake/external/pybind11.cmake
@@ -7,13 +7,14 @@ if(NOT TARGET pybind11::module)
   include(ExternalProject)
 
   set(pybind11_INCLUDE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/pybind11/src/pybind11/include)
-  set(pybind11_URL https://github.com/pybind/pybind11.git)
-  set(pybind11_TAG v2.6.2)
+  # set(pybind11_URL https://github.com/pybind/pybind11.git)
+  # set(pybind11_TAG v2.6.2)
 
   ExternalProject_Add(pybind11
         PREFIX pybind11
-        GIT_REPOSITORY ${pybind11_URL}
-        GIT_TAG ${pybind11_TAG}
+        # GIT_REPOSITORY ${pybind11_URL}
+        # GIT_TAG ${pybind11_TAG}
+        SOURCE_DIR @pybind11@
         CONFIGURE_COMMAND ""
         BUILD_COMMAND ""
         INSTALL_COMMAND ""
